src/spikeinterface/preprocessing/whiten.py

```diff
@@ -172,8 +197,26 @@ def compute_whitening_matrix(recording, mode, random_chunk_kwargs, apply_mean, r
         M = None
         data = random_data
 
-    cov = data.T @ data
-    cov = cov / data.shape[0]
+    if not regularize:
+        cov = data.T @ data
+        cov = cov / data.shape[0]
+    else:
+        import sklearn.covariance
+
+        if regularize_kwargs is None:
+            regularize_kwargs = {}
+        regularize_kwargs["assume_centered"] = True
+        job_kwargs = get_global_job_kwargs()
+        if "n_jobs" in job_kwargs and "n_jobs" not in regularize_kwargs:
+            n_jobs = job_kwargs["n_jobs"]
+            if isinstance(n_jobs, float) and 0 < n_jobs <= 1:
+                import os
+
+                n_jobs = int(n_jobs * os.cpu_count())
+            regularize_kwargs["n_jobs"] = n_jobs
+        estimator = sklearn.covariance.GraphicalLassoCV(**regularize_kwargs)
```

Comment:
maybe we could have a regularize method that can use lasso but maybe others no ?

Reply:
We could, but I would say let's start with one...

Reply:
I've tried something... We can pass method: "GraphicalLassoCV" or something else, and the appropriate node is created